verilog/tools/ls/autoexpand.cc

```diff
@@ -87,15 +87,16 @@ class AutoExpander {
 
   // Represents a port connection
   struct Connection {
-    absl::string_view port_name;  // The name of the port in the module instance
-    bool emit_dimensions;  // If true, when emitted, the connection should be
-                           // annotated with the signal dimensions
+    std::string port_name;  // The name of the port in the module instance
```

Comment:
Did you run into issues with the lifetime of strings, so you had to switch from string_view to std::string ?